apps/condo/domains/property/components/panels/Builder/BuildingPanelEdit.tsx

```diff
@@ -125,7 +139,7 @@ const ChessBoard: React.FC<IChessBoardProps> = ({ Builder, refresh }) => {
                             innerRef={container}
                         >
                             {
-                                Builder.visibleSections.length > 0 ? <BuildingAxisY floors={Builder.possibleFloors} /> : null
+                                Builder.visibleSections.length > 0 ? <BuildingAxisY floors={Builder.possibleChoosedFloors} /> : null
```

Comment:
I think it's better to name it **possibleChosenFloors**, but it's subjective and your call

Reply:
Done